### ISSUE TITLE
add "blank" music discs that can be burned with pv-addon-discs

### ISSUE THREAD

**Issue by ProjectHSI** [STEMMechanics/STEMCraft-Core#73]
basically a more achievable way of getting music discs to burn without too much effort. maybe something like they spawn in villages or can be craftable. the music discs by themselves can't be played but can be burned and then played with pv-addon-discs

**Comment by ProjectHSI** [comment]
code that deals with checking music discs for burn ellgibility
https://github.com/plasmoapp/pv-addon-discs/blob/5c59b459f39192c1372a61320b1887f2d00c0957/src/main/kotlin/su/plo/voice/discs/command/subcommand/BurnCommand.kt#L62

**Comment by ProjectHSI** [comment]
Eligibility is determined by an item's persistent data container.

https://github.com/plasmoapp/pv-addon-discs/blob/5c59b459f39192c1372a61320b1887f2d00c0957/src/main/kotlin/su/plo/voice/discs/DiscsPlugin.kt#L55

**Comment by ProjectHSI** [comment]
imagine NOT looking in the config file
couldn't be me

**Comment by nomadjimbob** [comment]
The idea of 'survival towns' is to have vendors (including the spawn town). We could have a vendor sell these blank discs and a second vendor which requests a url to burn onto the disc through chat?

**Comment by ProjectHSI** [comment]
2nd vendor idea sounds neat. Are you saying the 2nd vendor replaces the command or supplements it with an abstraction?

**Comment by nomadjimbob** [comment]
Replaces the command (runs the command on behalf of the player, or the plugin does what it needs to do to make it work). Mods would have the command to override if needed.

I started a SMShopKeeper Citizens trait in the 1.5 plugin but will prob be a 1.6 addition

**Comment by ProjectHSI** [comment]
Alright. I think we should have the command available for non-survival worlds (creative plots specifically)

**Comment by nomadjimbob** [comment]
Yep. We can enable it the command like we do with worldedit with `gamemode=creative`. Allows testing in the creative world, and mods to explicitly have to change gamemodes in survival worlds to use the command

**Comment by ProjectHSI** [comment]
Cost:
Bronze to buy a blank disc and erase a disc
Iron to burn a blank disc
